Resources/views/default.html.twig

```diff
@@ -2,7 +2,7 @@
     https://doc.ez.no/display/DOCTEST/Customer+Attribute+Mappings to see the list of default field names. #}
 
 <script id="recommendations-template-{{ templateId }}" type="text/x-handlebars-template">
-    {% raw %}
+    {% verbatim %}
```

Comment:
Actually we can drop Symfony 2.6 support now, as stated in https://github.com/ezsystems/EzSystemsRecommendationBundle/commit/7984abe4c9c54855bc5797e1fefd82eab08da0dd we don't use it in any activly supported releases anymore.